tests/test_crypto.py

```diff
@@ -971,6 +971,8 @@ def test_attributes(self):
         self.assertEqual(name.commonName, "quux")
         self.assertEqual(name.CN, "quux")
 
+        assert name.OU is None
+
```

Comment:
it would be nice if you could transform the rest too.  and add some empty lines to separate the blocks while doing so. :)

but i don’t want to see new code with old asserts :)